packages/falcon-sensor/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: https://github.com/CrowdStrike/falcon-helm/releases/download/0.9.3/falcon-sensor-0.9.3.tgz
-packageVersion: 00
+packageVersion: 01
```

Comment:
It looks like this PR only adds the display-name annotation to change the chart's name in the UI. If that's the case, then it might be better to remove this. This change will generate a new package that essentially has the same underlying chart as the previous version and could be confusing for users.

Reply:
That makes sense. Thanks @PennyScissors ! I have removed the package version bump.